apps/login-test-acceptance/tests/zitadel.ts

```diff
@@ -164,3 +169,22 @@ export function totp(secret: string) {
 
   return token;
 }
+
+export async function eventualNewUser(id: string) {
```

Comment:
The user should not be a problem, as there is a trigger on the Zitadel API, or am I wrong here?